app/models/public_body.rb

```diff
@@ -966,8 +953,7 @@ def name_for_search
 
   def self.get_public_body_list_translated_condition(table, has_first_letter=false, locale=nil)
     result = "(upper(#{table}.name) LIKE upper(:query)" \
-      " OR upper(#{table}.notes) LIKE upper(:query)" \
-        " OR upper(#{table}.short_name) LIKE upper(:query))"
+      " OR upper(#{table}.short_name) LIKE upper(:query))"
```

Comment:
Worth noting…

> It seems the searching of public bodies by their note content on `/body/list/all` isn't working which is why a spec has been disabled. Will open an issue to restore this functionality.

…in the commit message, just so if someone git-blames it's clear it was intentional and don't have to go back searching for the PR.